website/components/StyledLink/index.js

```diff
@@ -0,0 +1,12 @@
+import styled from 'styled-components';
+
+const StyledLink = styled.a`
+  color: ${(props) => (props.active ? props.theme.colors.primary600 : props.theme.colors.neutral600)};
```

Comment:
ℹ️ the `theme` value is directly passed by SC under the hood:

```js
color: ${({ theme, ...props }) => /* do stuff */}
```

Reply:
@Bassel17 I think you forgot this one!

`color: ${({ theme, active }) => (active ? theme.colors.primary600 : theme.colors.neutral600)};`